src/main/java/ua/edu/internship/interview/service/mapper/InterviewMapper.java

```diff
@@ -0,0 +1,23 @@
+package ua.edu.internship.interview.service.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.MappingTarget;
+import ua.edu.internship.interview.data.documents.InterviewDocument;
+import ua.edu.internship.interview.service.dto.interview.InterviewCreateDto;
+import ua.edu.internship.interview.service.dto.interview.InterviewDto;
+import ua.edu.internship.interview.service.dto.interview.InterviewUpdateDto;
+
+import java.util.List;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.List'
<!-- __reviewdog__:ChA3MmNiY2ViYWY0MjBmNTc0EgpjaGVja3N0eWxl -->